worker-markup-core/src/main/java/com/github/cafdataprocessing/worker/markup/core/EmailSplitter.java

```diff
@@ -53,7 +53,7 @@ public EmailSplitter(JepExecutor jepExec)
          * - "*-+[^-]+-+\s*" matches the actual divider text i.e. "---- abc ----"
          * - "$" makes sure the divider is at the end of the line.
          */
-        this.dividerPattern = Pattern.compile("(\n|^)(( |>)*-+[^-]+-+\\s*)$");
+        this.dividerPattern = Pattern.compile("(\n|^)(( |>)*+-++[^-]++-++\\s*+)$");
```

Comment:
Please update the comment above the regex to match the changes and describe why the possessive quantifiers are used here.